Sources/SecureStore/KeyManagerService.swift

```diff
@@ -96,10 +96,13 @@ extension KeyManagerService {
     // Retrieve a key that has been stored before
     func retrieveKeys(localAuthStrings: LocalAuthenticationLocalizedStrings? = nil) throws -> (publicKey: SecKey,
                                                                                                privateKey: SecKey) {
-        guard let privateKeyTag = "\(configuration.id)PrivateKey".data(using: .utf8) else {
+        let privateKeyTagData: Data? = Data("\(configuration.id)PrivateKey".utf8)
```

Comment:
There is no need to cast these as optionals.  This Data.init will never return nil, so you can get rid of the guard statements and thrown SecureStoreErrors completely.